sdk/storage/Azure.Storage.Files.Shares/src/Sas/ShareSasBuilder.cs

```diff
@@ -152,7 +152,7 @@ public ShareSasBuilder()
         /// class to create a Blob Service Sas.
         /// </summary>
         /// <param name="permissions">
-        /// The time at which the shared access signature becomes invalid.
+        /// This contains the list of permissions that can be set for a file's access policy.
```

Comment:
Can we change these to:

The list of permissions that can be set for a file's access policy.

Reply:
Just curious, why is the text for Azure Files different from what you used for blob, queue, and ADLS? The feature should behave the same across all of these services.